app/views/shared/_footer.html.haml

```diff
@@ -31,25 +31,26 @@
         // This is the instance-managed set of links:
         %h4
           = t '.footer_contact_headline'
-        %p.social-icons
-          - if ContentConfig.footer_facebook_url.present?
-            %a{href: ContentConfig.footer_facebook_url}
-              %i.ofn-i_044-facebook
-          - if ContentConfig.footer_twitter_url.present?
-            %a{href: ContentConfig.footer_twitter_url}
-              %i.ofn-i_041-twitter
-          - if ContentConfig.footer_instagram_url.present?
-            %a{href: ContentConfig.footer_instagram_url}
-              %i.ofn-i_043-instagram
-          - if ContentConfig.footer_linkedin_url.present?
-            %a{href: ContentConfig.footer_linkedin_url}
-              %i.ofn-i_042-linkedin
-          - if ContentConfig.footer_googleplus_url.present?
-            %a{href: ContentConfig.footer_googleplus_url}
-              %i.ofn-i_046-g
-          - if ContentConfig.footer_pinterest_url.present?
-            %a{href: ContentConfig.footer_pinterest_url}
-              %i.ofn-i_045-pintrest
+        - if ContentConfig.footer_facebook_url.present? || ContentConfig.footer_twitter_url.present? || ContentConfig.footer_instagram_url.present? || ContentConfig.footer_linkedin_url.present? || ContentConfig.footer_googleplus_url.present? || ContentConfig.footer_pinterest_url.present?
```

Comment:
we should rather extract a helper method from this long line and put it in `app/helpers/footer_links_helper.rb`.